packages/cactus-plugin-consortium-manual/README.md

```diff
@@ -0,0 +1,39 @@
+# `@hyperledger/cactus-plugin-consortium-manual`
+
+## Prometheus Exporter
+This class creates a prometheus exporter, which scrapes the total Cactus node count.
+
+### Usage Prometheus
+The prometheus exporter object is initialized in the `PluginConsortiumManual` class constructor itself, so instantiating the object of the `PluginConsortiumManual` class, gives access to the exporter object.
+You can also initialize the prometheus exporter object seperately and then pass it to the `IPluginConsortiumManualOptions` interface for `PluginConsortiumManual` constructor.
+
+`getPrometheusExporterMetricsV1` function returns the prometheus exporter metrics, currently displaying the total cactus node count, which currently increments everytime `updateMetricNodeCount` method of the `PluginConsortiumManual` class is called.
```

Comment:
> which currently increments everytime

@jagpreetsinghsasan I wouldn't say that it increments, I'd phrase it as "gets refreshed to match the node count in the consortium" or something like that, but incrementing makes it sound like we just keep increasing the node count even if there are no actual new nodes being added to the consortium for real.

Reply:
True. I will fix it asap.